src/test/java/bio/terra/service/snapshot/SnapshotDaoTest.java

```diff
@@ -267,7 +291,7 @@ public void snapshotEnumerateTest() throws Exception {
 
 
         MetadataEnumeration<SnapshotSummary> summaryEnum = snapshotDao.retrieveSnapshots(0, 2, null,
-            null, "==foo==", snapshotIdList);
+            null, "==foo==", null, snapshotIdList);
```

Comment:
Instead of modifying all these call sites, you could create an overloaded version of `SnapshotDao.retrieveSnapshots()` that forwards to the new version of the API. Although it looks like all the cases are in test code, so maybe it's not worth it.

Reply:
that was my thinking---and I want to stay consistent for now. I'm sure we will continue updating this endpoint